builtin/mainmenu/tab_settings.lua

```diff
@@ -263,7 +266,7 @@ local function handle_settings_buttons(this, fields, tabname, tabdata)
 		return true
 	end
 	if fields["cb_opaque_water"] then
-		core.settings:set("opaque_water", fields["cb_opaque_water"])
+		core.settings:set("opaque_water", bool_to_string[(not string_to_bool[fields["cb_opaque_water"]])])
```

Comment:
I think you're supposed to use `minetest.is_yes` instead of `string_to_bool`.

Reply:
I couldn't find that method in the (admittedly minimal) API research I did, but it _did_ work, so I've [committed](https://github.com/minetest/minetest/pull/12012/commits/34016d6692986fc718c70b26ac7a8d5f43d9ccce) appropriately. Thank you for the input!